dev/tasks/tasks.yml

```diff
@@ -1185,7 +1185,7 @@ tasks:
 {% endfor %}
 
 {% for turbodbc_version in ["latest", "master"] %}
-  test-conda-python-3.7-turbodbc-{{ turbodbc_version }}:
+  notest-conda-python-3.7-turbodbc-{{ turbodbc_version }}:
```

Comment:
@kszucs is there a better way to do this? I basically renamed this to "notest", so it won't be included in the `test-*` selection of the `nightly` group of tasks.

Reply:
You may comment out the yaml block so the glob pattern wouldn't pick it up.

Reply:
I was initially thinking that by keeping them uncommented you could still run those builds on demand (eg in a github comment), so therefore did it this way. But maybe that's not very useful anyway, so just commenting out might be the easier solution.

Reply:
Commented out instead